### PR TITLE
Added the always_run option in excepthook document page

### PR DESCRIPTION
**Pull request by tolgahanuzun**
https://github.com/getsentry/sentry-python/pull/364

Is there wrong or missing piece?

### CODE REVIEW

File: src/collections/_documentation/platforms/python/default-integrations.md [hunk 1]
```diff
@@ -23,6 +23,22 @@ This integration registers with the interpreter's except hook system.  Through t
...
 raised interactive interpreter sessions will not be reported.
 
+### Options
+
+You can pass the following keyword arguments to `ExcepthookIntegration()`:
+
+* `always_run`:
+    
+    ```python
+    #repl
```
Comment (untitaker) [inline src/collections/_documentation/platforms/python/default-integrations.md]: How about this?

```sh
$ python
>>> import sentry_sdk
>>> from sentry_sdk.integrations.excepthook import ExcepthookIntegration
>>> sentry_sdk.init(..., integrations=[ExcepthookIntegration(always_run=True)])
>>> raise Exception("I will become an error")
```

File: src/collections/_documentation/platforms/python/default-integrations.md [hunk 2]
```diff
@@ -23,6 +23,22 @@ This integration registers with the interpreter's except hook system.  Through t
...
+    raise Exception('I can catch an error through in Repl.')
+    ```
+    The Sentry does not catch error in repl when it is always_run's default value. If the default value is set to `True`, errors in repl can be captured. (`sentry_sdk.init(integrations=[ExcepthookIntegration(always_run=True)])`)
```
Comment (untitaker) [inline src/collections/_documentation/platforms/python/default-integrations.md]:
```suggestion
    By default, the SDK does not capture errors occuring in the REPL (`always_run=False`).
```

File: src/collections/_documentation/platforms/python/default-integrations.md (hunk 1 above)
Comment (tolgahanuzun) [reply src/collections/_documentation/platforms/python/default-integrations.md]: I'm not good at the document. This is much better. :) Thank you for both correction. I'm updating now.

**Comment by tolgahanuzun** [comment]
@untitaker anything else? :)

File: src/collections/_documentation/platforms/python/default-integrations.md [hunk 3]
```diff
@@ -23,6 +23,24 @@ This integration registers with the interpreter's except hook system.  Through t
...
+    ```bash
+    $ python
+    >>> import sentry_sdk
+    >>> from sentry_sdk.integrations.excepthook import ExcepthookIntegration
+    >>> sentry_sdk.init(..., integrations=[ExcepthookIntegration(always_run=True)])
+    >>> raise Exception("I will become an error")
+    ```
+    By default, the SDK does not capture errors occuring in the REPL (`always_run=False`).
+
+  The default is `False`.
```
Comment (untitaker) [inline src/collections/_documentation/platforms/python/default-integrations.md]: This line duplicates information. Also please check the indentation level, I think the code block is indented with three spaces

**Comment by tolgahanuzun** [comment]
> Sorry for the back-and-forth!

The learning process for me. I enjoy Review. :) I fixed it. @untitaker

**Comment by untitaker** [comment]
I will merge this once a new version is released, because this repo deploys on merge. I hope to get to it within this week! Thanks so far.

**Comment by untitaker** [comment]
@MimiDumpling looping you in based on the meeting we just had. I want to make sure we figure out a process that is not too much friction on external contributors.

File: src/collections/_documentation/platforms/python/default-integrations.md [hunk 4]
```diff
@@ -23,6 +23,22 @@ This integration registers with the interpreter's except hook system.  Through t
...
+    
+  ```bash
+  $ python
+  >>> import sentry_sdk
+  >>> from sentry_sdk.integrations.excepthook import ExcepthookIntegration
+  >>> sentry_sdk.init(..., integrations=[ExcepthookIntegration(always_run=True)])
+  >>> raise Exception("I will become an error")
+  ```
+
+  By default, the SDK does not capture errors occuring in the REPL (`always_run=False`).
```
Comment (MimiDumpling) [inline src/collections/_documentation/platforms/python/default-integrations.md]:
```suggestion
  By default, the SDK does not capture errors occurring in the REPL (`always_run=False`).
```

Comment (tolgahanuzun) [reply src/collections/_documentation/platforms/python/default-integrations.md]: ops, now fixing .. :) Bad typo

Comment (MimiDumpling) [reply src/collections/_documentation/platforms/python/default-integrations.md]: No worries! This looks great. :) Thanks for all your work!

**Comment by untitaker** [comment]
Thanks!

**Pull request merged**